release.py

```diff
@@ -168,12 +167,12 @@ def tweak_patchlevel(tag, done=False):
 
 /* Version as a string */
 #define PY_VERSION      \t\"{tag.text}{plus}"'''.strip()
-    level_def = dict(
-        a   = 'PY_RELEASE_LEVEL_ALPHA',
-        b   = 'PY_RELEASE_LEVEL_BETA',
-        rc  = 'PY_RELEASE_LEVEL_GAMMA',
-        f   = 'PY_RELEASE_LEVEL_FINAL',
-        )[tag.level]
+    level_def = {
+        'a': 'PY_RELEASE_LEVEL_ALPHA',
+        'b': 'PY_RELEASE_LEVEL_BETA',
+        'rc': 'PY_RELEASE_LEVEL_GAMMA',
+        'f': 'PY_RELEASE_LEVEL_FINAL',
+        }[tag.level]
```

Comment:
```suggestion
    level_def = {
        'a': 'PY_RELEASE_LEVEL_ALPHA',
        'b': 'PY_RELEASE_LEVEL_BETA',
        'rc': 'PY_RELEASE_LEVEL_GAMMA',
        'f': 'PY_RELEASE_LEVEL_FINAL',
    }[tag.level]
```
Shouldn't this line be dedented?  Maybe the linter gets confused by the `[tag.level]`?

Reply:
I would personally dedent it but the linter isn't bothered.

I avoided making any formatting changes here, we should just run Black on everything.